.spelling

```diff
@@ -562,6 +564,7 @@ Websockets
 WebSphere
 whitelist
 whitelists
+Wget
```

Comment:
Should `Wget` be allowed as the command is `wget`?

Reply:
@ericvn Note that the official name according to the webpage of the command is `Wget` https://www.gnu.org/software/wget/wget.html